app/util/data_preparation/dataframe_converter.py

```diff
@@ -0,0 +1,72 @@
+from decimal import Decimal, getcontext
```

Comment:
Can we move this file from this folder to`app/reports_generation/scipts`? 
This should simplify imports and makes all related to judgment code in one place.